Assets/Global/Scripts/Player/Player.cs

```diff
@@ -243,6 +250,16 @@ public void Heal(float reward)
 
     public void IncreaseMaxHealth(float reward) => playerStatistic.MaxHealth.AddModifier("reward", reward);
 
+    [ContextMenu("Add 30 crumbs")]
+    public void IncreaseCrumbs()
+    {
+        int crumbs = GlobalReference.PermanentPlayerStatistic.Get<int>("crumbs");
+        // GlobalReference.PermanentPlayerStatistic.Crumbs = crumbs + crumb;
+        GlobalReference.PermanentPlayerStatistic.Crumbs = crumbs + 30;
+        GlobalReference.PermanentPlayerStatistic.SaveCrumbs();
+        GlobalReference.AttemptInvoke(Events.CRUMBS_CHANGED);
```

Comment:
I would instead invoke this event in the SaveCrumbs method.
A lot more bug proof, and is less to remember